pkg/tiller/kind_sorter_test.go

```diff
@@ -50,6 +50,10 @@ func TestKindSorter(t *testing.T) {
 			Head: &util.SimpleHead{Kind: "Deployment"},
 		},
 		{
+			Name: "1",
```

Comment:
- [ ] Can we reshuffle the labels so that they come out in order on line 135?  
  This one would become 'f,' and the rest that follow would increase by one alphabetic position.
  It's inconvenient to do so, I know, but it makes it easier to figure out what should come before or after what.

Reply:
Yeah, I thought about that. In all honesty this should probably be reworked so we don't have to do it at all. Would that be a problem?

Reply:
Reworking it would be great. I went through and relabeled all of them when I added a bunch back during the Spring so that I could tell that the alphabetic sequence at the end was correct. Before then, the sequence at the end was random, but the single-character tags on the kinds were in order. I didn't find that helpful for determining where to put a new kind, nor did it help me spot problems in the order easily.

We have one sequence (really a set) of items, each with a label, and then two orderings of those labels. I ordered the kinds alphabetically, and assigned the labels in the intended ordering (see commit 7ccfc6d7d48c874bb5bd97624372e70bbba656c8). That doesn't accommodate insertions and deletions of kinds well.

Do you have a different idea that would be better?

Reply:
Do a strict array comparison of what is expected 😄